CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ The general format is:
 
 ```
 
+# 2.9.0 - DATE (04/08/2022)
```

Comment:
Can this be a PATCH since it's a bug fix? 🙂

Reply:
Sure thing! Thanks for the comment :)

Reply:
done